package.json

```diff
@@ -17,44 +17,44 @@
   ],
   "repository": "eclipse-velocitas/cli",
   "dependencies": {
-    "@oclif/core": "^2.9.3",
-    "@oclif/plugin-help": "^5.2.13",
-    "@oclif/plugin-not-found": "^2.3.30",
-    "@oclif/plugin-plugins": "^3.1.6",
+    "@oclif/core": "^3.12.0",
```

Comment:
Can we use a stable version - to determine our environment?

Reply:
will create another PR, dont want to update major versions now